src/dark.css

```diff
@@ -13,7 +13,7 @@
 
 .dark {
   background-color: #111;
-  color: #bbb;
+  color: #fff;
```

Comment:
i'm not too knowledgeable in this area, but maybe an opacity would make the white not be "too bright for dark mode"?

https://material.io/design/color/dark-theme.html#ui-application recommends 87% for high emphasis text. i think it could also apply to the grid?
![image](https://user-images.githubusercontent.com/1988030/147397599-f5f8fe08-8cde-4ffd-83e2-cb1b620edbfb.png)